packages/openmrs/src/http.js

```diff
@@ -0,0 +1,46 @@
+import { expandReferences } from '@openfn/language-common/util';
+import * as util from './Utils';
+
+/**
+ * Options object
+ * @typedef {Object} OpenMRSOptions
+ * @property {object} query - An object of query parameters to be encoded into the URL
+ * @property {object} headers - An object of all request headers
+ * @property {object} body - The request body (as JSON)
+ * @property {string} baseUrl - The base url for the request
+ * @property {string} [parseAs='json'] - The response format to parse (e.g., 'json', 'text', or 'stream')
+ */
+
+/**
+ * Make a HTTP request to any OpenMRS endpoint
+ * @example
+ * request("GET",
+ *   "/ws/rest/v1/patient/d3f7e1a8-0114-4de6-914b-41a11fc8a1a8", {
+ *    query:{ 
+ *       limit: 1, 
+ *       offset: 20 
+ *    },
+ *     baseUrl: "http://msf-ocg-openmrs3-dev.westeurope.cloudapp.azure.com/openmrs"
+ * });
+ * @function
+ * @public
+ * @param {string} method - HTTP method to use
+ * @param {string} path - Path to resource
+ * @param {OpenMRSOptions}  [options={}] - An object containing either query, headers, and body for the request
```

Comment:
Please remove the word "either", because it implies that only one of query, headers or body can be set